packages/xstate-react/test/useSpawn.test.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { useActor } from '../src';
+import { render, cleanup, fireEvent } from '@testing-library/react';
+import { useSpawn } from '../src/useSpawn';
+import { fromReducer } from 'xstate/src/behavior';
```

Comment:
I would export this from the core's index for now - so people won't have to reach for `/behavior`-named file.

Reply:
Ideally we would have `import { fromReducer } from 'xstate/behaviors'`, can we do this in v4?

Reply:
It's possible by adding an additional file (`packages/core/behaviors/package.json`) but I thought that we don't want to publicly call those behaviors in v4 and keep that as an implementation detail.

Reply:
We have to call this something:

```js
const actorRef = useSpawn(/* ??? */);
```

Reply:
Let's call this behaviors, I think it's useful to introduce this as a term.